site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,547 @@
+An introduction to Sphinx and Read the Docs for Technical Writers
+=================================================================
+
+By: Eric Holscher
+
+Bio: Eric Holscher is the co-founder of Read the Docs, and the Write the Docs conference.
+Along with community organizing and open source work,
+he does consulting around software documentation and speaks at a number of industry events each year.
+When he isn't expressing his views on software docs, he's getting views from the top of mountains.
+
+Treating documentation as code is becoming a major theme in the software industry.
+This is coming from both sides,
+with developers starting treat documentation as a priority alongside tests and code,
+and writers seeing a lot of value in integrating more into the development process.
+This marrying of cultures isn't simple,
+but having the proper tools for the job makes both sides happy with the results that get produced.
+
+A lot of developer tools don't work well for writers.
+Sphinx and Read the Docs unique in this ecosystem,
+in that they have powerful tools that both writers and developers want,
+in one convenient package.
+
+Overview of the Ecosystem
+-------------------------
+
+Read the Docs is the largest open source documentation hosting site in the world.
+Open Source in this context means both that the code is open source,
+and the documentation hosted is for open source software.
+It is a developer focused platform,
+but it has most of the features that technical writers have come to expect in a tool as well.
+This blending of worlds makes it the best suited platform for teams that want both writers and developers contributing to product and API documentation.
+
+Read the Docs is best thought of as a continuous documentation platform for Sphinx.
+Continuous documentation being similar to continuous integration in programming circles.
+In practice it means that your documentation is built,
+tested, 
+and updated on every commit.
+
+Sphinx provides a documentation generator that is best-in-class for software docs.
+Sphinx documents are written in the reStructuredText markup language.
+which is similar to Markdown,
+but much more powerful.
+Read the Docs provides a best-in-class hosting platform for Sphinx.
+
```

Comment:
that's an awful lot of "best-in-class"ness ;) -- although one instance is probably ok (I detest it, but it'll probably speak to an STC audience)